pkg/server/status.go

```diff
@@ -4211,7 +4211,7 @@ func (s *statusServer) localUpdateTableMetadataCache() (
 	select {
 	case s.updateTableMetadataJobSignal <- struct{}{}:
 	default:
-		return nil, status.Errorf(codes.Unavailable, "update table metadata cache job is not ready to start execution")
+		return nil, status.Errorf(codes.Aborted, "update table metadata cache job is not ready to start execution")
```

Comment:
Just curious, why the switch to aborted here?

Reply:
I think that typically, the unavailable status should be used for when the service is temporarily unavailable, where as the aborted status should be used when an operation is aborted. To me this case falls under an latter.